### PR TITLE
changed bootstrap design

### PR DESCRIPTION
**Pull request by Lasklu**
I changed the design preset with a custom bootstrap template and added some further changes

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
   postgres:
     image: postgres:13.1
     ports:
-      - 5432:5432
+      - 54322:5432
```
Comment (rohansaw) [inline docker-compose.yml:7]: why do we need this?

File: app/custom_bootstrap.css [hunk 2]
```diff
@@ -0,0 +1,7870 @@
+/* Created with Themestr.app */
+/*! `Custom` Bootstrap 4 theme */
```
Comment (rohansaw) [inline app/custom_bootstrap.css:2]: Cant this whole file be minified or couldn't we use a preprocessor (SCSS or SASS etc)

File: docker-compose.yml (hunk 1 above)
Comment (Lasklu) [reply docker-compose.yml:7]: no, we do not. Sorry I accidentally pushed my own setup. will change

File: app/custom_bootstrap.css (hunk 2 above)
Comment (Lasklu) [reply app/custom_bootstrap.css:2]: I will check

File: docker-compose.yml (hunk 1 above)
Comment (Lasklu) [reply docker-compose.yml:7]:
``` suggestion
      - 5432:5432
```

**Pull request closed without merging**